Launcher/Launcher/Launcher.Designer.vb

```diff
@@ -98,7 +98,7 @@ Partial Class frmLauncher
         Me.FormBorderStyle = System.Windows.Forms.FormBorderStyle.Fixed3D
         Me.MaximizeBox = False
         Me.Name = "frmLauncher"
-        Me.Text = frmLauncher_title
+        Me.Text = "OpenRCT2 Launcher"
```

Comment:
The titles should be edited in the resource files and not in the designer anymore.